sandbox-backend/src/main/java/com/exadel/sandbox/team5/dao/util/SearchCriteria.java

```diff
@@ -0,0 +1,30 @@
+package com.exadel.sandbox.team5.dao.util;
+
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Sort;
+
+import java.awt.print.Pageable;
+
+public class SearchCriteria {
+    private int page;
+    private int size;
+    private String properties;
```

Comment:
We should also have something like `List<Sorting>` instead of properties as one string value